ogip/spec.py

```diff
@@ -80,13 +80,38 @@ def from_file_name_osa(fn):
 
     @staticmethod
     def from_file_name_normal(fn):
+
         f = fits.open(fn)
+        if 'RATE' in f['SPECTRUM'].data.names:
+            rate = f['SPECTRUM'].data['RATE']
+        elif 'COUNTS' in f['SPECTRUM'].data.names:
+            rate = f['SPECTRUM'].data['COUNTS'] / f['SPECTRUM'].header['EXPOSURE']
```

Comment:
Header is asked for EXPOSURE many times making the lines longer than needed.

Maybe:

```suggestion
            rate = f['SPECTRUM'].data['COUNTS'] / exposure
```

and above 

```
exposure = f['SPECTRUM'].header['EXPOSURE']
```
?

Reply:
done